frontend/public/lib/locationOperations.ts

```diff
@@ -354,5 +354,6 @@ export async function getLocationFilteredBy(query) {
     return full_location;
   } catch (e) {
     console.log(e);
+    return null;
```

Comment:
changed return undefined to return null, if the backend fails to lookup the location by place_id.